CMakeLists.txt

```diff
@@ -557,6 +558,13 @@ IF(PLUS_USE_THORLABS_VIDEO)
   ENDIF()
 ENDIF()
 
+IF(PLUS_USE_OPENHAPTICS)
+  FIND_PACKAGE(OpenHaptics)
+  IF(NOT OPENHAPTICS_FOUND)

```

Comment:
Generally, with FindPackage modules, the case of the resulting variables should match the case of the Find argument.

In this case, `FIND_PACKAGE(OpenHaptics)` should produce variables of the form `OpenHaptics_`
or
`FIND_PACKAGE(OPENHAPTICS) --> OPENHAPTICS_`